lab1-review.c

```diff
@@ -0,0 +1,157 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <dirent.h>
+#include <unistd.h>
+#include <sys/stat.h>
+
+//Построчный вывод значений из аргументов командной строки
+void outputToScreen (int argc, char *argv[  ] ) {
+for(int i = 1; i < argc; i++) {
+printf ("%s\n", argv[i] );
+    }
+}
```

Comment:
Хуевое форматирование, нет табуляции и отступов

```suggestion
//Построчный вывод значений из аргументов командной строки
void outputToScreen (int argc, char *argv[  ] ) {
    for(int i = 1; i < argc; i++) {
        printf ("%s\n", argv[i] );
    }
}
```